### ISSUE TITLE
Change to Logout Spots way of working in-game.

### ISSUE THREAD

**Issue by yplusz** [MeteorDevelopment/meteor-client#691]
**Describe the feature**
Maybe instead of just a box that only spawns when the player is at render distance, you could also get a message through chat with who logged out + the coordenates of where it was.

**Comment by The-Warlord** [comment]
I'm 100% sure it's not possible to know where players logged out if they aren't in your render distance, as this would amount to a location hack, which would have been abused to hell long ago.

**Comment by yplusz** [comment]
> I'm 100% sure it's not possible to know where players logged out if they aren't in your render distance, as this would amount to a location/base finding hack, which would have been abused to hell long ago.

Yup, you're right! But I would really like the coords + nametag part, inertia has those but I rather use meteor, that's why I want it

**Comment by The-Warlord** [comment]
> > I'm 100% sure it's not possible to know where players logged out if they aren't in your render distance, as this would amount to a location/base finding hack, which would have been abused to hell long ago.
> 
> Yup, you're right! But I would really like the coords + nametag part, inertia has those but I rather use meteor, that's why I want it

Yes, nametags would be pretty pog.

**Comment by StormyBytes** [comment]
Logout spots already show the nametags.

**Comment by yplusz** [comment]
> > > I'm 100% sure it's not possible to know where players logged out if they aren't in your render distance, as this would amount to a location/base finding hack, which would have been abused to hell long ago.
> 
> > 
> 
> > Yup, you're right! But I would really like the coords + nametag part, inertia has those but I rather use meteor, that's why I want it
> 
> 
> 
> Yes, nametags would be pretty pog.

Absolutely pog.

**Comment by yplusz** [comment]
> Logout spots already show the nametags. 

You didn't get it, we are talking about the client ones, not just the plain name of the player.

**Comment by seasnail8169** [comment]
Log out spots does show "the client ones", some of the information cant be gotten about a play because they dont actually exist.

**Comment by yplusz** [comment]
> Log out spots does show "the client ones", some of the information cant be gotten about a play because they dont actually exist.

But then how Inertia one works? Not in any manner trying to criticize meteor's one, its just because its a pain to use both together...

**Comment by seasnail8169** [comment]
Ive never used inertias, what information does it show?

**Comment by yplusz** [comment]
> Ive never used inertias, what information does it show?

I can't provide a screenshot right now so imma describe it. It's basically the nametags module but with the logout spot. It shows exactly the same stuff, healt, gear, enchantments, name, player head, all of that. Tomorrow I can get a screenshot to show it if this description was confusing!

**Comment by seasnail8169** [comment]
ok sure, a screenshot would be nice

**Comment by yplusz** [comment]
> ok sure, a screenshot would be nice

that's okay! I can get one of those, may take a while, 9 hours more or less, imma add a screenshot to that my other post too, teleport finder.

**Comment by seasnail8169** [comment]
ok

**Comment by yplusz** [comment]
> ok

i couldn't get my own footage but ive found this: 
![inertia](https://user-images.githubusercontent.com/78128501/118838157-d650e700-b89b-11eb-80c4-33b2ed40038f.png)
it looks exactly like that, just as nametags module.

**Comment by seasnail8169** [comment]
Thats literally just nametags, i dont really understand what you want

**Comment by yplusz** [comment]
> Thats literally just nametags, i dont really understand what you want

yup! that's the point! i just want so that when the person logs out, the nametag is still there! that's just it

**Comment by seasnail8169** [comment]
Oh right, I guess I could do that

**Comment by yplusz** [comment]
> Oh right, I guess I could do that

Nice! Thank you very much!

**Comment by crosby-moe** [comment]
> Oh right, I guess I could do that

any updates?
